src/server/pfs/server/driver_v2.go

```diff
@@ -62,37 +63,57 @@ func newDriverV2(env *serviceenv.ServiceEnv, txnEnv *txnenv.TransactionEnv, etcd
 	if err != nil {
 		return nil, err
 	}
-	gcClient, err := gc.NewClient(db)
-	if err != nil {
-		return nil, err
-	}
+	tracker := tracker.NewPostgresTracker(db)
 	chunkStorageOpts, err := env.ChunkStorageOptions()
 	if err != nil {
 		return nil, err
 	}
-	chunkStorageOpts = append([]chunk.StorageOption{chunk.WithGarbageCollection(gcClient)}, chunkStorageOpts...)
-	d2.storage = fileset.NewStorage(objClient, chunk.NewStorage(objClient, chunkStorageOpts...), env.FileSetStorageOptions()...)
+	chunkStorage := chunk.NewStorage(objClient, chunk.NewPostgresStore(db), tracker, chunkStorageOpts...)
+	d2.storage = fileset.NewStorage(fileset.NewPostgresStore(db), tracker, chunkStorage, env.FileSetStorageOptions()...)
 	d2.compactionQueue, err = work.NewTaskQueue(context.Background(), d2.etcdClient, d2.prefix, storageTaskNamespace)
 	if err != nil {
 		return nil, err
 	}
-	go d2.master(env, objClient, db)
+	go d2.master(env)
 	go d2.compactionWorker()
 	return d2, nil
 }
 
-func newDB() (*gorm.DB, error) {
+func newDB() (db *sqlx.DB, retErr error) {
+	defer func() {
+		if db != nil {
+			db.MustExec(`DROP SCHEMA IF EXISTS storage CASCADE`)
+			fileset.SetupPostgresStore(db)
```

Comment:
It might be cleaner to just have all three of these be `SetupPostgres` and scope them through the package naming.

Reply:
Also it might make more sense to execute this if retErr == nil, rather than checking the db handle. Although I don't really think that would change the behavior as is, I think it is a better pattern to follow for potential changes in the future.

Reply:
All of this setup will change with the migrations stuff I'm working on.